roles/rsyslog/defaults/main.yml

```diff
@@ -14,8 +14,15 @@ rsyslog_package_name: rsyslog
 
 rsyslog_fluentd: true
 
-rsyslog_fluentd_port: 5140
 rsyslog_fluentd_host: 127.0.0.1
+rsyslog_fluentd_port: 5140
 
 fluentd_port: "{{ rsyslog_fluentd_port }}"
 fluentd_host: "{{ rsyslog_fluentd_host }}"
+
+##########################
+# additional rsyslog
+
+rsyslog_additional_protocol: udp
+rsyslog_additional_host: ""
```

Comment:
We just had a discussion in kolla in a similar situation. I think it is better to leave the variable undefined by default, the code does already check whether it is defined.

Reply:
CI is not happy with undefined variable because of the |length check in the template.

Reply:
hmm, o.k., let's leave it like this for now